README.md

```diff
@@ -1,47 +1,44 @@
-# Minetest Pick_Axe_Tweaks Mod
-
+# Minetest Pickaxe Tweaks Mod
 ![Luacheck](https://github.com/wsor4035/pick_axe_tweaks/workflows/luacheck/badge.svg)
 [![License: MIT](https://img.shields.io/badge/License-MIT-blue.svg)](https://opensource.org/licenses/MIT)
+[![ContentDB](https://content.minetest.net/wsor4035/pick_axe_tweaks/shields/downloads/)](https://content.minetest.net/packages/wsor4035/pick_axe_tweaks/)
 
 ## About
 
-adds the ability for pick axes to place a configurable light node ffrom in game and per player
+Adds the ability for pickaxes to place light nodes. Configurable per player.
 
 ## Links
 
 * [Github](https://github.com/wsor4035/pick_axe_tweaks)
-* [Contentdb](not yet available)
-* [forums](not avaible yet)
+* [ContentDB](https://content.minetest.net/packages/wsor4035/pick_axe_tweaks/)
+* Minetest forum (not avaible yet)
 
 ## Setup
 
-download mod, put in your worldmods folder, or mods folder and enable it
-* if not using with mtg, please change the setting found under header `minetest.conf settings`
-* also use a mod that uses this mods api to register its pick axes
+Download mod, put in your `mods` or `worldmods` folder and enable it.
+
+If not using with Minetest Game, please change the default light node setting.
```

Comment:
since its on contentdb/content tab, maybe mention that? feel free to ignore this comment as I myself just use content as a nice gui for finding mods git source